formats/json/jvmTest/src/kotlinx/serialization/json/GsonCompatibilityTest.kt

```diff
@@ -1,45 +1,56 @@
 package kotlinx.serialization.json
 
+import com.google.gson.*
 import kotlinx.serialization.*
-import kotlinx.serialization.descriptors.*
-import kotlinx.serialization.encoding.*
-import org.junit.*
+import org.junit.Test
+import kotlin.test.*
 
 class GsonCompatibilityTest {
 
-    @Serializable(with = ValueSerializer::class)
-    data class Value<T>(val isSet: Boolean, val value: T?)
```

Comment:
Is this test deleted at all?

Reply:
Nope, I've actually reverted what I've accidentally committed in the previous PR